base/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -21,56 +21,60 @@ spec:
         app: sourcegraph-frontend
     spec:
       containers:
-      - args:
-        - serve
-        env:
-        - name: PGDATABASE
-          value: sg
-        - name: PGHOST
-          value: pgsql
-        - name: PGPORT
-          value: "5432"
-        - name: PGSSLMODE
-          value: disable
-        - name: PGUSER
-          value: sg
-        - name: PUBLIC_REPO_REDIRECTS
-          value: "true"
-        - name: SRC_APP_URL
-          value: http://localhost:3080
-        - name: SRC_GIT_SERVERS
-          value: gitserver-1:3178
-        image: docker.sourcegraph.com/frontend:18579_2018-07-20_46e5ad9
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: http
-            scheme: HTTP
-          initialDelaySeconds: 5
-          timeoutSeconds: 5
-        name: frontend
-        ports:
-        - containerPort: 3080
-          name: http
-        - containerPort: 3090
-          name: http-internal
-        - containerPort: 3443
-          name: https
-        resources:
-          limits:
-            cpu: "2"
-            memory: 2G
-          requests:
-            cpu: "2"
-            memory: 1G
-        volumeMounts:
-        - mountPath: /etc/sourcegraph
-          name: sg-config
+        - args:
+            - serve
+          env:
+            - name: DEPLOY_TYPE
```

Comment:
shouldn't be necessary:
https://sourcegraph.sgdev.org/github.com/sourcegraph/sourcegraph/-/blob/pkg/conf/computed.go#L126:10